src/environment.js

```diff
@@ -289,18 +328,22 @@ const mdnPrototype = {
     /**
      * Accepts a relative URL or an attachment object
      * Returns the content of a given file.
+     *
+     * @param {string|{url:string}} fileObjOrUrl
      */
     async getFileContent(fileObjOrUrl) {
-        var url = fileObjOrUrl.url || fileObjOrUrl;
+        var url = typeof fileObjOrUrl === 'string'
+            ? fileObjOrUrl
+            : fileObjOrUrl.url;
         if (!url) return '';
 
         let base_url = '';
 
         // New file urls include attachment host, so we don't need to
         // prepend it
-        var fUrl = url.parse(url);
+        var fUrl = URL.parse(url);
         if (!fUrl.host) {
-            var p = url.parse(this.env.url, true);
+            var p = URL.parse(this.env.url, true);
```

Comment:
Previously, this would try&nbsp;to&nbsp;call `String.prototype.parse`, which doesn’t exist, causing a&nbsp;`TypeError` to&nbsp;be&nbsp;thrown.

I&nbsp;discovered this thanks&nbsp;to: https://medium.com/@trukrs/type-safe-javascript-with-jsdoc-7a2a63209b76.